### ISSUE TITLE
Searching feature explanation - New card creation (Link / Search / Template)

### ISSUE THREAD

**Issue by webenefits** [wekan/wekan#5550]
I just found a feature in WeKan that I'm not quite sure how it works, which is why I'm looking for documentation or an explanation. 

When a new card is created, several links are displayed below it:
![grafik](https://github.com/user-attachments/assets/e8aa2f48-72aa-49a5-b9df-95f4fe8360f1)

Is there any documentation for these functions? Unfortunately, I couldn't find anything in the wiki.

Many thanks in advance!

**Comment by xet7** [comment]
## Linked Cards

Linked card feature was added at https://github.com/wekan/wekan/pull/1592

With Linked Card feature, it's possible to create new board as dashboard, and link cards from other boards to current board, seeing realtimes changes from all those cards at dashboard.

Search is for searching card, to be as linked card, I presume.

There is also Linked Board, but I'm not sure is it useful.

## Templates

Template is about Board/List/Card templates, that are at right top username / Templates. But it does not work well yet.

https://github.com/wekan/wekan/wiki/Templates

**Comment by xet7** [comment]
I added wiki page for this to https://github.com/wekan/wekan/wiki/Linked-Cards

**Comment by webenefits** [comment]
Ah great, thank you very much!

**Comment by ITT5** [comment]
Wow - what a great feature! Needed and found it today. One question:

- I created a card on board A. On this board I am the only member.
- I linked this card to board B. On this board there are other members as well.
- Now I want to add some of those members of board B to the linked card.
- This does not work. I could add members to this "mothercard" in board A but I am not able to add members to the linked card in board B.

I am using Edge Browser in current version.
